.release-please-manifest-submodules.json

```diff
@@ -73,6 +73,7 @@
     "metastore": "1.12.0",
     "migrationcenter": "0.1.0",
     "monitoring": "1.15.1",
+    "netapp": "0.0.0",
```

Comment:
Just an FYI the post processor is should be smart enough to add the RP files these days: https://github.com/googleapis/google-cloud-go/blob/e3de5f89b6f255c37eb4323775ee3b59093428cf/internal/postprocessor/releaseplease.go#L122

https://github.com/googleapis/google-cloud-go/blob/e3de5f89b6f255c37eb4323775ee3b59093428cf/internal/postprocessor/releaseplease.go#L72-L93

Reply:
@quartzmo

Reply:
Good to know, thank you!